mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
```

Comment:
The variable `deletedOrder` is not checked for null before checking its truthiness. It should be checked for null or undefined before checking its truthiness to avoid potential null pointer exceptions.